### PR TITLE
Add analysis intro with TMO

### PR DESCRIPTION
**Pull request by fbertsch**
Open to any and all changes.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/firefox-data-docs/28)

### CODE REVIEW

**Comment by harterrt** [comment]
Looks like there's a merge conflict.

**Comment by fbertsch** [comment]
@harterrt RFAL

File: concepts/analysis_intro.md [hunk 1]
```diff
@@ -1,6 +1,55 @@
...
+
+This article is under development.
+The work is being tracked in
+link:https://bugzilla.mozilla.org/show_bug.cgi?id=1341706[this bug].
+
+= Getting Started with Firefox Data
+// Introduces:
+//   * Pings
+
+Firefox clients out in the wild send us data as *pings*. Most of our pings contain some combination of *environment* data (e.g. operating system, hardware, Firefox version) and *probe* data (e.g. max number of open tabs, time spent running in Javascript garbage collection). We have quite a few different pings, but most of our data for Firefox Desktop comes in from link:http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/main-ping.html[Main pings].
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Isn't all data "probe" data? Can we change this to "behavior" data?

File: concepts/analysis_intro.md [hunk 2]
```diff
@@ -1,6 +1,55 @@
...
 [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341706).
 
+=======
+= Work in Progress
+
+This article is under development.
+The work is being tracked in
+link:https://bugzilla.mozilla.org/show_bug.cgi?id=1341706[this bug].
+
+= Getting Started with Firefox Data
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Looks like this markup is still in Asciidoc, but the file name is `.md`. Feel free to choose either, but it needs to be consistent. I personally prefer markdown.

File: concepts/analysis_intro.md [hunk 3]
```diff
@@ -1,6 +1,55 @@
...
+// Introduces:
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Asciidoc markup.

File: concepts/analysis_intro.md [hunk 4]
```diff
@@ -1,6 +1,55 @@
-# Work in Progress
-
 This article is under development.
-The work is being tracked in 
+The work is being tracked in
 [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341706).
 
+=======
+= Work in Progress
```
Comment (harterrt) [inline concepts/analysis_intro.md]: The work in progress note should be removed with this commit.

File: concepts/analysis_intro.md (hunk 1 above)
Comment (harterrt) [inline concepts/analysis_intro.md]: The link here is Asciidoc markup.

File: concepts/analysis_intro.md [hunk 5]
```diff
@@ -1,6 +1,55 @@
...
+link:https://bugzilla.mozilla.org/show_bug.cgi?id=1341706[this bug].
+
+= Getting Started with Firefox Data
+// Introduces:
+//   * Pings
+
+Firefox clients out in the wild send us data as *pings*. Most of our pings contain some combination of *environment* data (e.g. operating system, hardware, Firefox version) and *probe* data (e.g. max number of open tabs, time spent running in Javascript garbage collection). We have quite a few different pings, but most of our data for Firefox Desktop comes in from link:http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/main-ping.html[Main pings].
+
+== Probes
+// Introduces:
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Asciidoc markup.

File: concepts/analysis_intro.md [hunk 6]
```diff
@@ -1,6 +1,55 @@
...
+//   * Histograms
+//   * Scalars
+
+When we need to measure specific things about clients, we use probes. A single ping will send in many different probes. There are two types of probes: *Histograms* and *Scalars*.
```
Comment (harterrt) [inline concepts/analysis_intro.md]: I think of probes as a tool to collect additional data, e.g. the code used to populate a field with a measurement. I would consider *Histograms* and *Scalars* to be data types.

File: concepts/analysis_intro.md [hunk 7]
```diff
@@ -1,6 +1,55 @@
...
+
+Histograms are bucketed counts. The link:https://github.com/mozilla/gecko-dev/blob/master/toolkit/components/telemetry/Histograms.json[Histograms.json] file has the definitions for all histograms, which includes the minimum, maximum, and number of buckets. Any recorded value instead just increments it's associated bucket. We have four main types of histograms:
```
Comment (harterrt) [inline concepts/analysis_intro.md]: The link uses Asciidoc markup.

File: concepts/analysis_intro.md [hunk 8]
```diff
@@ -1,6 +1,55 @@
...
+1. Linear - Buckets are divided evenly between the minimum and maximum; e.g. [1-2] is a bucket, and so is [100-101].
+2. Boolean - Only two buckets, associated with true and false.
+3. Enumerated - Integer buckets, where usually each bucket has a label.
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Can you list 'linear' and 'exponential' histograms next to each other? Maybe:
* Boolean
* Enumerated
* Linear
* Exponential

File: concepts/analysis_intro.md [hunk 9]
```diff
@@ -1,6 +1,55 @@
...
+4. Exponential - Larger valued buckets cover a larger range; e.g. [1-2] is a bucket, and so is [100-200].
+
+To see some of these in action, take a look at the link:https://telemetry.mozilla.org/histogram-simulator[Histogram Simulator].
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Asciidoc markup.

File: concepts/analysis_intro.md [hunk 10]
```diff
@@ -1,6 +1,55 @@
...
+
+Scalars are simply a single value. The link:https://dxr.mozilla.org/mozilla-central/rev/tip/toolkit/components/telemetry/Scalars.yaml[Scalars.yaml] file has the definitions for all scalars. These values can be integers, strings, or booleans.
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Asciidoc markup.

File: concepts/analysis_intro.md [hunk 11]
```diff
@@ -1,6 +1,55 @@
...
+
+== TMO
+The simplest way to start looking at probe data is in link:https://telemetry.mozilla.org/new-pipeline/dist.html[the Measurement Dashboard] or link:https://telemetry.mozilla.org/new-pipeline/evo.html[the Distribution Dashboard]. Using these dashboards you can compare a probes value between populations, e.g. GC_MS for 64 bit vs. 32 bit, and even track it across builds.
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Asciidoc markup.

File: concepts/analysis_intro.md [hunk 12]
```diff
@@ -1,6 +1,55 @@
...
+
+The Measurement Dashboard is a snapshot, aggregating all the data from all chosen dimensions. The Y axis is % of samples, and the X axis is the bucket. You can compare between dimensions, but it does not give you the ability to see how data is changing over time. To investigate that you must use the Evolution Dashboard.
+
+The Evolution Dashboard shows how the data changes over time. Choose which statistics you'd like to plot over time, e.g. Median or 95th percentile. The Y axis is time, and the X axis is the value for whichever statistic you've chosen. link:https://telemetry.mozilla.org/new-pipeline/evo.html#!aggregates=median!95th-percentile&cumulative=0&end_date=2017-06-13&keys=!__none__!__none__&max_channel_version=nightly%252F56&measure=GC_MS&min_channel_version=nightly%252F53&processType=*&product=Firefox&sanitize=1&sort_keys=submissions&start_date=2017-06-12&trim=1&use_submission_date=0[This dashboard], for example, shows how GC_MS is improving from nightly 53 to nightly 56! While the median is not changing much, the 95th percentile is trending down, indicating that long garbage collections are being shortened.
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Asciidoc markup.

File: concepts/analysis_intro.md [hunk 13]
```diff
@@ -1,6 +1,55 @@
...
+
+The X axis on the Evolution Dashboard shows either Build ID (a date), or Submission Date. The difference is that on any single date we might recieve submissions from lots of builds, but aggregating by Build ID means we can be sure a change was because of a new build.
+
+The second plot on the Evolution View is the number of pings we saw containing that probe (Metric Count).
+
+=== TMO Caveats
+* Data is aggregated on a per-ping basis, meaning *these dashboards cannot be used to say something definitive about users*. Please repeat that to yourself. A trend in the evolution view may be caused not by a change affecting lots of users, but a change affecting one user who is now sending 50% of the pings we see. link:http://reports.telemetry.mozilla.org/post/projects%2Fproblematic_client.kp[And yes, that does happen.]
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Asciidoc markup.

Comment (harterrt) [inline concepts/analysis_intro.md]: Lol @ "please repeat that to yourself". Great warning :)

File: concepts/analysis_intro.md [hunk 14]
```diff
@@ -1,6 +1,55 @@
...
+* Sometimes it looks like no data is there, but you think there should be. Check under advanced settings and check "Don't Sanitize" and "Don't Trim Buckets". If it's still not there, let us know in IRC on #telemetry.
+
+== Where to go next
+* Analysis using STMO
+* Advanced analysis with ATMO
```
Comment (harterrt) [inline concepts/analysis_intro.md]: Should these be links?

**Comment by fbertsch** [comment]
@harterrt RFAL

File: concepts/analysis_intro.md [hunk 15]
```diff
@@ -1,6 +1,43 @@
-# Work in Progress
+Getting Started with Firefox Data
+=================================
 
-This article is under development.
-The work is being tracked in 
-[this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341706).
+Firefox clients out in the wild send us data as *pings*. [http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/main-ping.html](Main pings) contain some combination of *environment* data (e.g. operating system, hardware, Firefox version), *measurements* (e.g. max number of open tabs, time spent running in Javascript garbage collection), and *events* (a work in progress). We have quite a few different pings, but most of our data for Firefox Desktop comes in from main pings.
```
Comment (harterrt) [inline concepts/analysis_intro.md]: It looks like these links are reversed. Syntax for a link is `[text](url)` but these appear to be `[url](text)`. You can verify that these links render correctly by running `gitbook serve` locally.

Please fix here and throughout.

File: concepts/analysis_intro.md [hunk 16]
```diff
@@ -1,6 +1,43 @@
...
+### TMO Caveats
+* Data is aggregated on a per-ping basis, meaning *these dashboards cannot be used to say something definitive about users*. Please repeat that to yourself. A trend in the evolution view may be caused not by a change affecting lots of users, but a change affecting one user who is now sending 50% of the pings we see. [http://reports.telemetry.mozilla.org/post/projects%2Fproblematic_client.kp](And yes, that does happen.)
+* Sometimes it looks like no data is there, but you think there should be. Check under advanced settings and check "Don't Sanitize" and "Don't Trim Buckets". If it's still not there, let us know in IRC on #telemetry.
+
+Where to go next
+----------------
+* [Analysis using STMO](tools/stmo.md)
+* [Advanced analysis with ATMO](tools/spark.md)
+* [Experimental data](tools/experiments.md)
+* [Adding probes, collecting more data](https://developer.mozilla.org/en-US/docs/Mozilla/Performance/Adding_a_new_Telemetry_prob://developer.mozilla.org/en-US/docs/Mozilla/Performance/Adding_a_new_Telemetry_probe)
```
Comment (harterrt) [inline concepts/analysis_intro.md]: This page does not load for me.

File: concepts/analysis_intro.md [hunk 17]
```diff
@@ -1,6 +1,43 @@
...
+
+The second plot on the Evolution View is the number of pings we saw containing that probe (Metric Count).
+
+### TMO Caveats
+* Data is aggregated on a per-ping basis, meaning *these dashboards cannot be used to say something definitive about users*. Please repeat that to yourself. A trend in the evolution view may be caused not by a change affecting lots of users, but a change affecting one user who is now sending 50% of the pings we see. [http://reports.telemetry.mozilla.org/post/projects%2Fproblematic_client.kp](And yes, that does happen.)
+* Sometimes it looks like no data is there, but you think there should be. Check under advanced settings and check "Don't Sanitize" and "Don't Trim Buckets". If it's still not there, let us know in IRC on #telemetry.
+
+Where to go next
+----------------
+* [Analysis using STMO](tools/stmo.md)
```
Comment (harterrt) [inline concepts/analysis_intro.md]: This relative link points to `/concepts/tools/stmo.md`. Please add a leading `/`, here and below, making the URL: `/tools/stmo.md`

**Comment by fbertsch** [comment]
@harterrt rookie mistakes :) RFAL

**Comment by harterrt** [comment]
LGTM! Thanks!

**Pull request merged**